src/tool_doswin.c

```diff
@@ -714,6 +714,8 @@ static struct TerminalSettings {
 #define ENABLE_VIRTUAL_TERMINAL_PROCESSING 0x0004
 #endif
 
+bool tool_isVirtualTerminal;
```

Comment:
Why the `virtual` part of the name? The bold is output when the terminal is real as your screenshot shows. The virtual part puzzles me.

Reply:
Users have good reasons to turn VTS off even in Win-10 build 16299.
It simply means `(TerminalSettings.dwOutputMode & ENABLE_VIRTUAL_TERMINAL_PROCESSING) == TRUE`.

Reply:
Okay, that's the technical name for the feature in the OS but for people reading the code, wouldn't a better name be something like `terminal_has_bold_enabled` ? It is the implied connection between "virtual" and "it can do styled output" in the variable name that I reacted to. Not that the the check if the styled output works actually needs to check with something that is called a virtual terminal.

Reply:
Ok I changed the name to `tool_term_has_bold`